lm-feedCore-iOS/lm-feedCore-iOS/Source/Classes/Screens/TaggingListView/LMFeedTaggingListViewModel.swift

```diff
@@ -29,7 +29,7 @@ public final class LMFeedTaggingListViewModel {
     // MARK: Init
     init(delegate: LMFeedTaggingListViewModelProtocol?) {
         self.currentPage = 1
-        self.pageSize = 10
+        self.pageSize = 20
```

Comment:
Was it intentional or was for testing purpose ?

Reply:
It was intentional